packages/ng-schematics/test/src/e2e.test.ts

```diff
@@ -6,19 +8,19 @@ import {
   setupHttpHooks,
 } from './utils.js';
 
-describe('@puppeteer/ng-schematics: e2e', () => {
+void describe('@puppeteer/ng-schematics: e2e', () => {
```

Comment:
do we need void test promises?

Reply:
We have `@typescript-eslint/no-floating-promises` so EsLint will fail. Sadly there is not a way to provide a Ignore Pattern to said rule.

Reply:
Failed updating to ESM as some Testing from Angular is only CJS